CONTRIBUTING.md

```diff
@@ -1,90 +1,69 @@
-
-## Development on Windows
-
-### Clone repo
-
-Clone the repository to your local machine from the Azure DevOps endpoint.
-
-### Build and Test
-
-**Note:** You will need NodeJS to be able to build the project and download dependencies. Go [here](https://nodejs.org/en/download/) to download NodeJS.
-
-**Note:** The project uses webpack for module bundling. Refer to the webpack documentation [here](https://webpack.js.org/).
+Retrospectives is an Azure DevOps extension. Visit
+[this link](https://docs.microsoft.com/en-us/azure/devops/extend/?view=vsts) to
+learn more about developing extensions. The Retrospectives Azure DevOps
+extension frontend is implemented in React using Typescript, and the backend is
+implemented in C#. The project follows a single branch source control strategy.
 
-- Clone this repo and open it with Visual Studio or with Visual Studio Code. You can also use any IDE or editor that you normally use to work with C# and React/Typescript.
+## Contributing Guidelines
 
-- Using Powershell, navigate to the '/RetrospectiveExtension.Frontend' folder, run `npm install`. This will download all the dependent packages listed in 'package.json'.
+### Branching and Pull Requests
 
-- When developing or publishing the extension locally, you need to create a `.env` file at the top level directory of the front end project (where `package.json` lives). You can copy `RetrospectiveExtension.Frontend/.env.template` to `RetrospectiveExtension.Frontend/.env` to get started. The contents of the `.env` file are
+1. When creating a new branch, follow the `{alias}/{issue##}` naming
+convention.
+    - **Note:** It is recommended to keep the branch name length below 30
+    characters in order to allow the Github Action which builds and deploys the
+    development extension on Pull Request creation or update to successfully execute.
 
-```bash
-# Backend Service URL
-REACT_APP_COLLABORATION_STATE_SERVICE_URL="put the deployed backend service url here"
-# App Instrumentation Key
-REACT_APP_APP_INSIGHTS_INSTRUMENTATION_KEY="put Instrumentation key here"
-```
+2. Once your feature addition or bug fix is ready for review, create a pull
+request against the `master` branch of the repository.
 
-- In lieu of the `.env` file you can set actual environment variables.
-- When using the CI/CD github action(s) pipeline to deploy the extension, environment variables are used to set Application Insights instrumentation key and the backend service url.
+3. Include a link to the bug or task you are addressing to the description of
+your pull request. Reviewers will be added to the pull request automatically.
 
-- Run `npm run build` to build the project. Refer to the 'scripts' section in 'package.json' for other commands.
+4. Ensure builds are successful and tests, including any added or updated tests
+, pass prior to submitting the pull request.
```

Comment:
Maybe an explicit reminder of the `npm run test` here?

Reply:
I was thinking about that - however I think that the pre-commit hook runs tests automatically so ff a user has gotten to the point where they can create a new PR with pushed code, local tests will have succeeded. Maybe this should be more of a reminder to make sure that all of the checks on GitHub pass